queue/driver_async_test.go

```diff
@@ -0,0 +1,288 @@
+package queue
+
+import (
+	"context"
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/suite"
+
+	"github.com/goravel/framework/contracts/queue"
+	configmock "github.com/goravel/framework/mocks/config"
+	ormmock "github.com/goravel/framework/mocks/database/orm"
+	queuemock "github.com/goravel/framework/mocks/queue"
+)
+
+var (
+	testAsyncJob       = 0
+	testDelayAsyncJob  = 0
+	testCustomAsyncJob = 0
+	testErrorAsyncJob  = 0
+	testChainAsyncJob  = 0
+)
+
+type DriverAsyncTestSuite struct {
+	suite.Suite
+	app        *Application
+	mockConfig *configmock.Config
+	mockQueue  *queuemock.Queue
+}
+
+func TestDriverAsyncTestSuite(t *testing.T) {
+	mockConfig := &configmock.Config{}
+	mockQueue := &queuemock.Queue{}
```

Comment:
Use the `New*` method, `AssertExpectations` doesn't need to be called.

```suggestion
	mockConfig := configmock.NewConfig(t)
	mockQueue := queuemock.NewQueue(t)
```